tests/migrations/test_graph.py

```diff
@@ -178,28 +176,25 @@ def test_graph_recursive(self):
         backwards_plan = graph.backwards_plan(root)
         self.assertEqual(expected[::-1], backwards_plan)
 
-    def test_graph_iterative(self):
+    def test_iterative_dfs_complexity(self):
+        n = 300
         graph = MigrationGraph()
-        root = ("app_a", "1")
-        graph.add_node(root, None)
-        expected = [root]
-        for i in range(2, 1000):
-            parent = ("app_a", str(i - 1))
-            child = ("app_a", str(i))
-            graph.add_node(child, None)
-            graph.add_dependency(str(i), child, parent)
-            expected.append(child)
-        leaf = expected[-1]
-
-        with self.assertWarnsMessage(RuntimeWarning, RECURSION_DEPTH_WARNING):
-            forwards_plan = graph.forwards_plan(leaf)
-
-        self.assertEqual(expected, forwards_plan)
-
-        with self.assertWarnsMessage(RuntimeWarning, RECURSION_DEPTH_WARNING):
-            backwards_plan = graph.backwards_plan(root)
-
-        self.assertEqual(expected[::-1], backwards_plan)
+        for i in range(1, n + 1):
+            graph.add_node(("app_a", str(i)), None)
```

Comment:
Use single quotes for strings, unless there's a nested single quote. ([Python style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style))

Reply:
I used `"app_a"` since the entire file uses double quotes.

Should I change to single quotes just the lines that I modify? Or the entire file?